### DISCUSSION TITLE
Using GitHub support for auto-generated release notes

### DISCUSSION THREAD

**Discussion by sugatoray** [omry/omegaconf#796] [Ideas]
Currently `towncrier` is used to publish release notes for `omegaconf`. But very recently GitHub made an announcement about the availability of a feature to produce auto-generated release notes. The short video demos look quite intriguing. The GitHub blog-post and related links are shared below. Perhaps, if this is considered useful, at a later stage this feature could be adopted for omegaconf.

- [x] [A new public beta of GitHub Releases: _How we’re improving the release experience_](https://github.blog/2021-10-04-beta-github-releases-improving-release-experience/)
  - [x] [Introducing auto-generated release notes](https://github.blog/2021-10-04-beta-github-releases-improving-release-experience/#introducing-auto-generated-release-notes)
  - [x] [A complete UI refresh for Releases](https://github.blog/2021-10-04-beta-github-releases-improving-release-experience/#a-complete-ui-refresh-for-releases)

> [LinkedIn post by GitHub](https://www.linkedin.com/posts/github_auto-generated-release-notes-activity-6850866677786267648-4qgX)

cc: @omry @Jasha10 @odelalleau

**Comment by omry** [comment]
Thanks @sugatoray. For now we are happy with towncrier but we will consider it.
